src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowPredicate.java

```diff
@@ -402,6 +402,9 @@ private static boolean getPredicateForAwaitingResponsesNonFullDefenceReceived(Ca
     public static final Predicate<CaseData> takenOfflineBySystem = caseData ->
         caseData.getTakenOfflineDate() != null;
 
+    public static final Predicate<CaseData> takenOfflineAfterSDO = caseData ->
```

Comment:
Does this work without causing ambiguous transitions? I would have expected it being the same as takenOfflineBySystem to cause problems potentially - but if it works it works!

Reply:
takenOfflineBySystem  changes/checks different date in caseData than this one.

Reply:
It looks like the same one to me:
takenOfflineBySystem --> caseData.getTakenOfflineDate()
takenOfflineAfterSdo  --> caseData.getTakenOfflineDate()

But I'll trust your judgement.